readme.md

```diff
@@ -216,10 +216,9 @@ Delays between retries counts with function `1000 * Math.pow(2, retry) + Math.ra
 Option `retries` can be a `number`, but also accepts a `function` with `retry` and `error` arguments. Function must return delay in milliseconds (`0` return value cancels retry).
 
 **Note:** It retries only on the specified methods, status codes, and on these network errors:
-- `ETIMEDOUT`: Connection was not estabilished after a period of time.
+- `ETIMEDOUT`: One of the [timeout](#timeout) limits was breached.
```

Comment:
Wouldn't be this better:
> One of the [timeouts](#timeout) has occured.

I don't think `breach` is a good word here to describe that.